packages/protocol/contracts/identity/Random.sol

```diff
@@ -13,7 +13,18 @@ contract Random is IRandom {
 
   bytes32 public _random;
 
+  uint256 public randomnessBlockRetentionWindow = 256;
+
+  mapping (uint256 => bytes32) private history;
+  uint256 private historyFirst;
+  uint256 private historySize;
+
   function initialize() external {
+    randomnessBlockRetentionWindow = 256;
```

Comment:
Should the initial value here be an argument to `initialize` instead? Would be more consistent with our current approach of storing config in `migrationsConfig.js`